embassy-stm32/Cargo.toml

```diff
@@ -71,7 +71,7 @@ futures-util = { version = "0.3.30", default-features = false }
 rand_core = "0.6.3"
 sdio-host = "0.5.0"
 critical-section = "1.1"
-stm32-metapac = { path = "../../stm32-data/build/stm32-metapac" }
+stm32-metapac = { version = "15.1001.0", registry = "artifactory" }
```

Comment:
I don't think this is a good idea. I think you should merge in the version number for the version you merged and then, once my change is merged (with the new version number), I'll come back and touch this file again to make it use the new one.

The idea here is that changes should be made in steps and that way we can isolate problems with `git bisect` more readily as they occur. The crates won't overwrite each other if they have different versions, so this means we'll be able to integrate and test your changes and **then** my changes. If we were to bump the version here, we would be mixing the two, and therefore it will make it more difficult to figure out the cause of any issues we might see.